### PR TITLE
INT B-20935

### PR DESCRIPTION
**Pull request by danieljordan-caci**
## Summary

`os.system` = bad
`subprocess.run` = mo bettuh

`subprocess.run()`: From what I've read, this function allows you to pass the command and arguments as a list, so no shell is invoked. This approach is safer than os.system() because it doesn’t mess with special shell characters (like &, ;, etc.), which could lead to command injections.

Added in `check=True` because this flag makes `subprocess.run()` raise an exception if the command fails, which can be helpful for debugging.

### How to test

1. Not much to test, just ensure everything builds and passes successfully

### CODE REVIEW

**Review by lucyluuuu — APPROVED** [review]
Documentations states subprocess.run is the recommended way to execute external shell commands/ system commands and check=True takes care of error handling.

**Comment by danieljordan-caci** [comment]
> Documentations states subprocess.run is the recommended way to execute external shell commands/ system commands and check=True takes care of error handling.

Thank you!!

**Review by traskowskycaci — APPROVED** [review]
Looks good!

File: pkg/gen/ghcapi/ghcoperations/queues/get_moves_queue_parameters.go [hunk 1]
```diff
@@ -574,7 +574,7 @@ func (o *GetMovesQueueParams) bindSort(rawData []string, hasKey bool, formats st
 // validateSort carries on validations for parameter Sort
 func (o *GetMovesQueueParams) validateSort(formats strfmt.Registry) error {
 
-	if err := validate.EnumCase("sort", "query", *o.Sort, []interface{}{"customerName", "dodID", "emplid", "branch", "locator", "status", "originDutyLocation", "destinationDutyLocation", "requestedMoveDate", "appearedInTooAt", "assignedTo", "counselingOffice"}, true); err != nil {
+	if err := validate.EnumCase("sort", "query", *o.Sort, []interface{}{"customerName", "edipi", "emplid", "branch", "locator", "status", "originDutyLocation", "destinationDutyLocation", "requestedMoveDate", "appearedInTooAt", "assignedTo", "counselingOffice"}, true); err != nil {
```
Comment (deandreJones) [inline pkg/gen/ghcapi/ghcoperations/queues/get_moves_queue_parameters.go:577]: what's going on here

**Pull request merged**